### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by kotalwar-krishna**
- css related to textutilities you can find in text-utilities.css
**screenshots**
txt1
txt2
txt3

### CODE REVIEW

File: documentation/doc.html [hunk 1]
```diff
@@ -410,8 +410,7 @@ <h4 class="h4-heading">This is heading 4</h4>
                 <h5 class="h5-heading">This is heading 5</h5>
               </div>
               <div>
-                <iframe src="https://www.thiscodeworks.com/embed/6203b444234b8a0015700cf5" style="width: 100%; height: 164px;" frameborder="0"></iframe>            </div>
-
+                <iframe src="https://www.thiscodeworks.com/embed/6203b444234b8a0015700cf5" style="width: 100%; height: 229px;" frameborder="0"></iframe>
```
Comment (Bhakti199) [inline documentation/doc.html:413]: use rem instead of px

**Review by Bhakti199 — APPROVED** [review]
good to merge

File: documentation/doc.html (hunk 1 above)
Comment (kotalwar-krishna) [reply documentation/doc.html:413]: ok

**Review by paarthbajaj — APPROVED** [review]
Seems good.

**Pull request merged**